examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,1234 @@
+{
```

Comment:
<p>Can we have all imports in the beginning of the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>absolutely, moved this to the beginning</p>